<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="model.syncManager.MessageSyncManagerImplTest" tests="4" skipped="0" failures="0" errors="0" timestamp="2017-11-18T18:01:56" hostname="raphael-MS-7816" time="0.076">
  <properties/>
  <testcase name="testA" classname="model.syncManager.MessageSyncManagerImplTest" time="0.011"/>
  <testcase name="testB" classname="model.syncManager.MessageSyncManagerImplTest" time="0.024"/>
  <testcase name="testC" classname="model.syncManager.MessageSyncManagerImplTest" time="0.017"/>
  <testcase name="testD" classname="model.syncManager.MessageSyncManagerImplTest" time="0.024"/>
  <system-out><![CDATA[javax.validation.ConstraintViolationException: Bean Validation constraint(s) violated while executing Automatic Bean Validation on callback event:'prePersist'. Please refer to embedded ConstraintViolations for details.
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
